.NET/Microsoft.Recognizers.Text.DateTime/Japanese/Parsers/TimePeriodFunctions.cs

```diff
@@ -0,0 +1,166 @@
+﻿using System;
```

Comment:
The same, I'm not quite sure whether we should put this file under "Parsers" folder or not. @tellarin

Reply:
I agree with Mu. They seem generic and not Japanese-specific. Maybe in /Utilities?